src/foam/u2/Element.js

```diff
@@ -2177,7 +2177,7 @@ foam.CLASS({
       name: 'createMode',
       documentation: 'The display mode for this property when the controller mode is CREATE.',
       assertValue: function(newValue) {
-        foam.assert(newValue === foam.u2.DisplayMode.RW || newValue === foam.u2.DisplayMode.HIDDEN, `Create mode must be RW or HIDDEN. Property '${this.of ? this.of.id + '.' : ''}${this.name}' was '${newValue.label}' and must be fixed.`);
+        foam.assert(newValue === foam.u2.DisplayMode.RW || newValue === foam.u2.DisplayMode.HIDDEN || newValue === foam.u2.DisplayMode.RO, `Create mode must be RW or HIDDEN or RO. Property '${this.of ? this.of.id + '.' : ''}${this.name}' was '${newValue.label}' and must be fixed.`);
```

Comment:
Wouldn't it be simpler to just say that we don't support DISABLED?

Reply:
Also, add a comment explaining why DISABLED isn't supported for create mode.

Reply:
Ya that makes sense 🤔

Reply:
Just talked with Anna, and I think it makes sense to just remove the `assertValue` for `createMode`. I can see why RO could make sense if you want to show a dynamically updating field in CREATE mode based on another property, so that makes sense. I still can't think of a reason someone would want a DISABLED input in create mode, but I don't think there's a good reason for the framework to prevent anyone from doing that if they want to. I think the `assertValue` on `readMode` still makes sense though.

Reply:
I agree with @TW80000. Let's just remove.